delta/commands.adoc

```diff
@@ -1361,5 +1361,10 @@ We can nest composite commands arbitrarily.{fn-org360}
 .Technical name
 `CompositeCommand`
 
+****
```

Comment:
I'm not sure we want this in the spec document.
According to https://diataxis.fr/, specs should not contain explanations.
I tend to put footnotes on these things with links to the explanations / rationale.

Reply:
I think a GH issue is just a bit too far removed. Given that this side note resolves potential confusion with the spec, I could even argue that it should be an actual part of it (regardless of https://diataxis.fr). So, I opt for a footnote.